samples/sensor/ltc4162/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _ltc4162:
+
+LTC4162: Step-Down battery charger
+##################################
+
+Description
+***********
+
+This sample application retrieves all the charger parameters,
+- Charger Status
+- Charger Type
+- Charger Health
+- Input Voltage in Volt
+- Input Current in Amps
+- Constant Current in Amps
+- Constant Current Max in Amps
+- Constant Voltage in Volt
+- Constant Voltage Max in Volt
+- Input Current limit in Amps
+- Charger die temperature in Celcius
+
+from LTC4162 sensor and print this information to the UART console.
+
+Requirements
+************
+
+This sample uses the LTC4162 charger controllered by using the I2C interface.
+
+References
+**********
+
+ - LTC4162S - https://www.analog.com/media/en/technical-documentation/data-sheets/LTC4162-S.pdf
+
+Building and Running
+********************
+
+ It requires a LTC4162S charger controller. It should work with any platforms featuring a
+ I2C peripheral interface. It does not work on QEMU. In this example below the
```

Comment:
The same issues are in this README as we previously discussed in another PR: https://github.com/zephyrproject-rtos/zephyr/pull/31801#discussion_r672613376.